### DISCUSSION TITLE
How does one set the retraction distance in Orca?

### DISCUSSION THREAD

**Discussion by dniezby1** [OrcaSlicer/OrcaSlicer#971] [General]
I've found the setting for retraction speed but not the distance. I'm coming from Cura so I'm assuming there is a different naming scheme for it in Orca?

I'm using Orca 1.6.2

**Comment by TechDude23** [comment]
Hello,

It's in the printer settings also in the filament setting overrides.  

**Travel distance threshold**  = When to wait for a retraction after so much distance.

You also have the retraction **length** but I think you are wanting the travel distance ?

**Comment by gasprints** [comment]
TechDude is correct.  Here is a screenshot to help you locate where the settings are
<img width="1185" alt="Screenshot 2023-05-08 at 4 52 54 PM" src="https://user-images.githubusercontent.com/128255447/236961383-5e170a75-dc5b-4075-8fc4-3463808f5e90.png">
